src/Bridge/Symfony/DependencyInjection/SonataTwigExtension.php

```diff
@@ -43,21 +43,22 @@ public function load(array $configs, ContainerBuilder $container): void
      */
     public function registerFlashTypes(ContainerBuilder $container, array $config): void
     {
+        // NEXT_MAJOR: change types to string[]
         $mergedConfig = array_merge_recursive($config['flashmessage'], [
             'success' => ['types' => [
-                'success' => ['domain' => 'SonataTwigBundle'],
```

Comment:
I'm not quite sure, but this looks like a BC break

Reply:
Can you read this? https://github.com/sonata-project/twig-extensions/pull/90#issuecomment-644382673